build-tools/packages/version-tools/test/schemes.test.ts

```diff
@@ -51,3 +86,35 @@ describe("detectVersionScheme", () => {
         assert.strictEqual(detectVersionScheme(input), expected);
     });
 });
+
+describe("getLatestReleaseFromList", () => {
+    const versionList: string[] = [];
+    versionList.push(...pre1VersionList, ...post1VersionList);
+
+    it("detects 1.2.2 is latest release", () => {
+        const expected = "1.2.2";
+        const latest = getLatestReleaseFromList(versionList);
+        assert.strictEqual(latest, expected);
+    });
+
+    it("detects 1.2.3 is latest release", () => {
+        const expected = "1.2.3";
+        versionList.push(expected);
+        const latest = getLatestReleaseFromList(versionList);
+        assert.strictEqual(latest, expected);
+    });
+
+    it("detects 2.0.0-internal.1.0.1 is latest release", () => {
+        const expected = "2.0.0-internal.1.0.1";
+        versionList.push(expected);
+        const latest = getLatestReleaseFromList(versionList);
+        assert.strictEqual(latest, expected);
+    });
+
+    it("detects 0.59.4002 is latest release", () => {
+        const expected = "0.59.4002";
+        versionList.push(expected);
```

Comment:
Probably don't need this here.

Reply:
This is to test the case where only pre-v1 versions are in the list. It shouldn't be a different result, but I want to be thorough. Added some detail to the test description.